appium/webdriver/webdriver.py

```diff
@@ -361,6 +362,56 @@ def find_elements_by_android_viewtag(self, tag):
         """
         return self.find_elements(by=MobileBy.ANDROID_VIEWTAG, value=tag)
 
+    def find_element_by_android_data_matcher(self, name=None, args=None, className=None):
```

Comment:
I would say it would be better to move these two methods into a separate extension